### DISCUSSION TITLE
Welcome to PAODiscussions!

### DISCUSSION THREAD

**Discussion by whart222** [or-fusion/pao#72] [General]
## 👋 Welcome!
  We’re using Discussions to provide a context for:
  * Asking questions about PAO capabilities.
  * Share ideas.
  * Engage with other community members.
  * Learning about PAO development plans.

**Comment by jaafaramine** [comment]
hello , I have quadratic bilevel optimization with bilinear terms in the objective function , how can I express them in my code.
